### PR TITLE
docs/readme: Add Bugzilla issue reporting link

### PR DESCRIPTION
**Pull request by ironcladlou**
Bugs are tracked in Bugzilla. GitHub issues are disabled for the repository, so make a note in the README to redirect users to Bugzilla.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,8 @@ The operator tries to be useful out of the box by creating a working default dep
 ## How to help
 
 See [HACKING.md](HACKING.md) for development topics.
+
+
+## Reporting issues
+
+Bugs are tracked in [Bugzilla](https://bugzilla.redhat.com/enter_bug.cgi?product=OpenShift%20Container%20Platform).
```
Comment (Miciah) [inline README.md]: How about preselecting the component and version? https://bugzilla.redhat.com/enter_bug.cgi?product=OpenShift%20Container%20Platform&version=4.0.0&component=Routing

**Comment by ironcladlou** [comment]
@Miciah Fixed/squashed, PTAL

**Pull request merged**